src/libraries/System.Private.CoreLib/src/System/Net/WebUtility.cs

```diff
@@ -804,6 +804,10 @@ static HtmlEntities()
             {
                 // Make sure the initial capacity for s_lookupTable is correct
                 Debug.Assert(s_lookupTable.Count == Count, $"There should be {Count} HTML entities, but {nameof(s_lookupTable)} has {s_lookupTable.Count} of them.");
+
+                // Just a quick check precalculated values in s_lookupTable are correct
+                Debug.Assert(ToUInt64Key("quot") == 0x71756F74);
+                Debug.Assert(ToUInt64Key("diams") == 0x6469616D73);
             }
```

Comment:
If you typed one of the hash codes wrong, or an entry was missed from the table, then I assume this test would fail?
https://github.com/dotnet/runtime/blob/e6f093cf0e711d9ba91b133b23a38c9a1911584d/src/libraries/System.Web.HttpUtility/tests/HttpUtility/HttpUtilityTest.cs#L95

I compared all the entities in that test vs this table, and they match except for one case: `&apos;` is missing from the test, but it is in the table. You might consider adding to that test.

Reply:
so those are just quick debug-time tests, but as you pointed out - the libs tests cover all the possible inputs (I added the missing one)